WAW.API/Chat/Resources/ChatRoomRequest.cs

```diff
@@ -0,0 +1,14 @@
+using System.ComponentModel.DataAnnotations;
+using Swashbuckle.AspNetCore.Annotations;
+
+namespace WAW.API.Chat.Resources;
+
+public class ChatRoomRequest {
+  [SwaggerSchema("Chat room creation date", Nullable = false)]
+  [Required]
+  public DateTime CreationDate { get; set; }
+
+  [SwaggerSchema("Chat room last update date", Nullable = false)]
+  [Required]
+  public DateTime LastUpdateDate { get; set; }
```

Comment:
These ones should be auto populated by the service. The only thing you should get from a `ChatRoomRequest` should be the list of participants.